packages/ffe-core-react/src/typography/LinkText.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { bool, node, string } from 'prop-types';
+import classNames from 'classnames';
+
+const LinkText = (props) => {
+    const {
+        children,
+        className,
+        href,
+        underline,
+        ...rest
+    } = props;
+
+    return (
+        <a
+            className={classNames(
+                'ffe-link-text',
+                { 'ffe-link-text--no-underline': !underline },
+                className,
+            )}
+            href={href}
+            {...rest}
+        >
+            {children}
```

Comment:
You don't have to destructure `children` here - it can be spread with the `rest` prop.